robot.py

```diff
@@ -45,6 +46,7 @@ class MyRobot(magicbot.MagicRobot):
     spinner: Spinner
     turret: Turret
     led_screen: LEDScreen
+    range_finder: RangeFinder
```

Comment:
Don't you want this to get its distance before other things run so they actually read an up-to-date distance?